fluster/decoders/gstreamer.py

```diff
@@ -900,3 +900,15 @@ def __init__(self) -> None:
         self.api = "SW"
         self.caps = self.caps + ",format=S16LE"
         super().__init__()
+
+
+@register_decoder
+class FluendoLCEVCHWVAH264Decoder(GStreamer10Video):
+    """LCEVC-H264 decoder for GStreamer 1.0"""
+
+    codec = Codec.H264
+    decoder_bin = "flulcevchwvah264dec"
```

Comment:
Does this element have hwva264dec decoder embedded or should the pipeline be something similar to
```suggestion
    decoder_bin = "flulcevchwvah264dec ! fluhwvah264dec"
```